From bae5eed03d025256d098f5fd9204bcfc58bc5452 Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Date: Thu, 30 Jan 2020 16:49:36 +0200
Subject: [PATCH 075/269] s32g274a: Define HW_ASSISTED_COHERENCY

We are using Ncore, so give this hint to bl31_plat_enable_mmu().
It is safe to do so, as the secondary cores only get to execute that
function after NCore has been set up for their cluster.

Upstream-Status: Pending 

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
Issue: ALB-4497
---
 plat/s32g/platform.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/plat/s32g/platform.mk b/plat/s32g/platform.mk
index ad919bbdc..b26f98b8c 100644
--- a/plat/s32g/platform.mk
+++ b/plat/s32g/platform.mk
@@ -66,6 +66,8 @@ RESET_TO_BL31			:= 0
 # We need SMP boot in order to make specific initializations such as
 # secure GIC registers, which U-Boot and then Linux won't be able to.
 COLD_BOOT_SINGLE_CPU		:= 0
+# Using Ncore as the cache coherency interconnect
+HW_ASSISTED_COHERENCY		:= 1
 
 ### Platform-specific defines ###
 # Which LinFlexD to use as a UART device
-- 
2.17.1

